allofplos/article_class.py

```diff
@@ -635,12 +642,12 @@ def get_contributors_info(self):
                 else:
                     print('one_corr_author error finding email for {} in {}'.format(corr_author, email_dict))
                     matching_error = True
-        elif len(corr_author_list) > 1 and len(set([tuple(x) for x in email_dict.values()])) > 1:
+        elif email_dict and len(corr_author_list) > 1 and len(set([tuple(x) for x in email_dict.values()])) > 1:
```

Comment:
This and the below check for email_dict being None aren't necessary with #41's fixes